library-backend/library-app/src/main/java/org/internship/library/app/security/MyUserDetailsService.java

```diff
@@ -50,6 +52,7 @@ public UserEntity signUp(UserEntity user) {
         PasswordValidationHelper.validatePassword(user);
         String encodedPassword = applicationPasswordEncoder.encode(user.getPassword());
         user.setPassword(encodedPassword);
+        user.setUserRole(USER);
```

Comment:
Can you use the role received in the UserEntity parameter instead of hardcoding it?